stdlib/http/src/main/java/org/ballerinalang/net/http/WebSocketConstants.java

```diff
@@ -83,6 +75,8 @@
     public static final int STATUS_CODE_ABNORMAL_CLOSURE = 1006;
     public static final int STATUS_CODE_FOR_NO_STATUS_CODE_PRESENT = 1005;
 
+    public static final int DEFAULT_MAX_FRAME_SIZE = 65536;
```

Comment:
We don't need this constant right? We have set the default value in bal file as well. Having two values for the same configuration from different places is not good IMO.

Reply:
It is used in the java level to set the size to default value if the value set by user is negative.